frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -594,4 +594,15 @@ common_settings.screenshot = {
     keep_menu_open = true,
 }
 
+common_settings.metric_system = {
+    text = _("Metric system"),
+    checked_func = function()
+        return G_reader_settings:readSetting("metric_system", true)
+    end,
```

Comment:
Do we really have to have a setting for this ? for the 2 nearly hidden places that show some size ?
I'd say that if I don't know how long is a "inch", for lack of ever seeing this unit, I guess all our users over the world (but may be not their blind grandma) should have a grasp of mm, cm, meters and understand what it means/sizes when seeing it).

Reply:
I can comment out the menu entry. We can delete it if nobody requests it in  lets say two releases :)
(As a physicist the Système international d’unité is the inly one to take care of :) )

Reply:
'Murica!

(Unfortunately, yeah, it might make sense to have a toggle available somewhere ;o).

Reply:
Let's @Frenzie have the final word.
(If kept, may be move the menu item nearer to the other unit one/ones like 12/24h.)

Reply:
We should have a setting for it, unless we just show both. Default to metric please. ;-)

Reply:
Yepp, default to metric is clear.
Should I put the menu entry into "Time and Date" (as the last entry there) and rename that entry to "Time, date and units"?